mono/metadata/w32process-unix.c

```diff
@@ -3644,6 +3644,38 @@ big_up (gconstpointer datablock, guint32 size)
 }
 #endif
 
+gboolean
+mono_w32process_time_date_stamp (gunichar2 *filename, guint32 *out)
```

Comment:
This `w32process` files are _only_ useful for supporting the managed `System.Diagnostics.Process` and we want to avoid adding new dependencies on that. Whatever you need to do here, *since it's not used by managed `Process`*, needs to go in https://github.com/mono/mono/blob/master/mono/utils/mono-proclib.c or in `eglib`.